orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/ClientSecretDaoImpl.java

```diff
@@ -91,4 +94,23 @@ public boolean setAsPrimary(ClientSecretEntity clientSecret) {
         query.setParameter("clientSecret", clientSecret.getClientSecret());
         return query.executeUpdate() > 0;
     }
+    
+    @Override
+    @Transactional
+    public List<ClientSecretEntity> getNonPrimaryKeys() {
+        DateTime dt = DateTime.now();
```

Comment:
Hey @auumgn, you are not updating this dt, so, it keeps being now(), please update it so it is now() - 24 hours before assign it to the "yesterday" variable.

Thanks